package/src/Camera.tsx

```diff
@@ -470,6 +470,7 @@ export class Camera extends React.PureComponent<CameraProps, CameraState> {
     }
 
     const shouldEnableBufferCompression = props.video === true && frameProcessor == null
+    const pixelFormat = props.pixelFormat ?? (frameProcessor == null ? 'yuv' : 'native')
```

Comment:
shouldn't this be the other way around? Unless i'm missing something this sets 'yuv' if there is no frame processor